### ISSUE TITLE
CyTube Logo Proposal v.1.0

### ISSUE THREAD

**Issue by christopherkc** [calzoneman/sync#473]
<b> Updated Tab Version </b>
<b> Design based on Favicon </b>
<b> File type: PNG  </b>

<b> Description:</b> 
Initially a logo based on the favicon. Was later redesigned more creatively using the whole tab as a logo including a new font and background. Stands out from all other logos on the market in general and one very appealing 'tab' to bookmark.

![cytubetab2](https://cloud.githubusercontent.com/assets/12298384/7573992/f9f37b0c-f7f7-11e4-98dc-7c4b4e9a763e.png)

**Comment by christopherkc** [comment]
<b> Airlock Version </b>
<b> Design based on Website Name </b>
<b> File type: PNG  </b>

<b> Description:</b> 
Website name used as a logo in Airlock font. Has a tube like rectangle with granulated texture at the end. 

![cytubev3 0](https://cloud.githubusercontent.com/assets/12298384/7555129/1bd5aa6c-f710-11e4-8ad4-275aa3a103e2.png)

**Comment by nuclearace** [comment]
But why.

**Comment by christopherkc** [comment]
It's a showcase for future reference - should the logo be changed later on.

![capture](https://cloud.githubusercontent.com/assets/12298384/7555106/4b136f22-f70f-11e4-91a5-2fd397e2e4a6.JPG)

**Comment by christopherkc** [comment]
<b> Futuristic Version </b>
<b> Design based on Identity. </b>
<b> File type: PNG  </b>

<b> Description: </b>
A motto integrated with the logo. Helps visitors and new users grasp the idea around what the website is and what it features. A very neat design.

![cytube4 0](https://cloud.githubusercontent.com/assets/12298384/7572857/46852618-f7ef-11e4-8d8a-3c9d69fcc75f.png)

**Comment by ghost** [comment]
what if I don't want to call my cytube sever cytube?

**Comment by christopherkc** [comment]
The default name / logo can always be changed as it is right now. Snoopy's channel is an example. Here's a screenshot.

![channel](https://cloud.githubusercontent.com/assets/12298384/7575950/49784e5a-f807-11e4-901a-dca6890bafec.JPG)

Channel link: http://cytu.be/r/Snoopy

**Comment by christopherkc** [comment]
Here is another example.

![capture](https://cloud.githubusercontent.com/assets/12298384/7576705/c8734af0-f80e-11e4-896f-d0425a09199d.JPG)

Channel link: http://cytu.be/r/Heinrich

**Comment by ghost** [comment]
How does one even set the logo?

**Comment by christopherkc** [comment]
Good question. There's several ways to go about it involving different types of codes. What the website currently has is a navbar-brand. "Navbars are responsive 'meta' components that serve as navigation headers for your application or site. Navbars collapse in mobile views and become horizontal as the available viewport width increases. At its core, the navbar includes styling for site names and basic navigation." Based on online info it would be written as something along these lines. See navbar-brand line.

Code reference: http://www.bootply.com/103980

You'll notice it's written with a url link, so the logo would have to be uploaded to a media-link site first - an extra step. 

Otherwise the way I've done it before and has worked is by inserting a rollover image with home page link (in html - nav section) and uploading it from the desktop. The image size and position won't be a problem for mobile sites either.

![rolloverimgcode](https://cloud.githubusercontent.com/assets/12298384/7578438/ffcfb678-f822-11e4-9a0a-5f7888dce1ca.JPG)

**Comment by christopherkc** [comment]
Also in reference to the question: "what if I don't want to call my cytube sever cytube?". The way it would work is to only update the main page i.e. reception area of the website which only takes 5 seconds to do if the code is properly written out. Then leave channel sections as is. So absolutely nothing in the channels would be changed.

If later on developers want to add the new logo to the channels (just hypothetically speaking) they can always add it <i> somewhere else </i> other than the upper top left section. 

Center of navigation bar, next to the copyright notice at the bottom, pretty much anywhere.

So in short even in that context admin users would still be able to use their own names / logos.

**Comment by Xaekai** [comment]
Channel owners can already set this to something custom if they feel so inclined, so I don't see why the site needs anything more elaborate. You also have to consider any changes like this have to look good on every theme. Seems way more work than its worth.

**Comment by christopherkc** [comment]
Yes, as I've already mentioned above and put up screenshots channel owners can already set something custom. If you read the thread carefully having the logo in channels hasn't been in the plan, <i> it was only as reference to Bush 6's question - we're talking about the home page. </i> As for theme matching, black and white matches a whole spectrum of colours that's why the majority of websites have either or both of those two colors. The logo takes 5 seconds to put up which is pretty much one of the fastest easiest thing one can set up - see screenshots above.

**Comment by christopherkc** [comment]
I think we need to keep in mind the bigger picture as well. A logo isn't just a designer feature it's also about functions it carries out, it's as important as the theme, it's the representation of what CyTube is all about, it's identity, it's goal, ambition, sense of direction. The opportunity to <i> first impress </i> someone new (could be just about anyone) enough for them to learn more about the website and service, go beyond just the home page, bookmark it, refer it to someone else. But even more than that, a great logo that would stand out in a positive way from other similar video syncing websites. 

![synaptop_logo_dark](https://cloud.githubusercontent.com/assets/12298384/7693788/0ad9182a-fda5-11e4-9282-408923bb8c7a.png)

![capturerabbit](https://cloud.githubusercontent.com/assets/12298384/7693791/238f1dba-fda5-11e4-969e-dfb269cefdc1.JPG)

![capture](https://cloud.githubusercontent.com/assets/12298384/7555106/4b136f22-f70f-11e4-91a5-2fd397e2e4a6.JPG)

Awhile ago, I had designed a website for a client and I had really thought it was all set and done, this weekend she e-mailed me with an urgent tag line that she needed a company logo uploaded to the site. Why? I have no clue but it must have been a pretty good reason if it had an urgent tag line.

In any case, we'll leave this thread for calzone to review as reference or whenever it is needed in the future.

**Comment by Xaekai** [comment]
CyTube sells itself by being better than the alternatives. It doesn't need to shill itself with a lot of glitz to hide shit code underneath. It's not SoundCloud.

**Comment by christopherkc** [comment]
You'd be surprised at how many people statistically don't even know what CyTube is, google "watch videos with friends" and see how CyTube ranks in terms of SEO. It's like trying to sell Chia seeds over milk, people have to actually know of it for it to be better. Also, it's not a matter of just back-end code sooner or later there will be emerging websites with coding features just as great, and in the end it'll come down to overall presentation as well as UX, UI, when there's a tie in terms of code.
